.vscode/settings.json

```diff
@@ -1,5 +1,7 @@
 {
-  "editor.rulers": [80],
+  "editor.rulers": [
+    80
```

Comment:
Unrelated change, please revert.

Reply:
@bajtos The format is changed when auto save. And if you check the other settings, breaking it into multiple the lines is the expected format.

I can try to move the change into a separate commit.

Reply:
> The format is changed when auto save. And if you check the other settings, breaking it into multiple the lines is the expected format.

Fair enough :+1:

> I can try to move the change into a separate commit.

Yes please. A new PR would be even better, that way we can land this small fix sooner.

Reply:
It turns out it's just my vscode's weird settings.
The other settings are broken down into multiple lines due to the max characters per line.
Reverting.